src/test/java/org/scijava/module/ModuleServiceTest.java

```diff
@@ -158,6 +159,66 @@ public void testGetSingleInput() throws ModuleException {
 		assertSame(info.getInput("double2"), singleDouble);
 	}
 
+	@Test
+	public void testSaveAndLoad() {
+
+		List<Object[]> parameterizations = new ArrayList<>();
+		parameterizations.add(new Object[] { //
+			double[].class, new double[] {} //
+		});
+		parameterizations.add(new Object[] { //
+			double[].class, new double[] { 1., 2., 3. } //
+		});
+		parameterizations.add(new Object[] { //
+			Double[].class, new Double[] {} //
+		});
+		parameterizations.add(new Object[] { //
+			Double[].class, new Double[] { 1., 2., 3. } //
+		});
+
+		for (Object[] params : parameterizations) {
```

Comment:
It looks like `params[0] == params[1].getClass()` for every element. So then, can't this be simplified to:
```java
List<Object> parameterizations = new ArrayList<>();
parameterizations.add(new double[] {});
parameterizations.add(new double[] {1., 2., 3.});
parameterizations.add(new Double[] {});
parameterizations.add(new Double[] {1., 2., 3.});
```
and then:
```java
Object expected = params;
Class<?> type = expected.getClass();
```
?

And maybe change the name from `parameterizations` to just `objects`?

Reply:
Yes, all good revisions. Just fixed them.